src/test/groovy/com/github/vkorobkov/jfixtures/loader/RowTest.groovy

```diff
@@ -94,4 +94,43 @@ class RowTest extends Specification {
         expect:
         EqualsVerifier.forClass(Row).verify()
     }
+
+    def "#columns returns row from key/value pairs"() {
+        given:
+        def row = new Row("vlad", columns)
+
+        when:
+        def result = row.columns(
+                "name", "Vlad",
+                "age", 30,
+                "hobby", "sleep"
+        )
+
+        then:
+        result.columns == [id: Value.of(1), name: Value.of("Vlad"), age: Value.of(30), hobby: Value.of("sleep")]
+    }
+
+    def "#columns throws IllegalArgumentException when odd number of key/value pairs"() {
+        given:
+        def row = new Row("vlad", columns)
+
+        when:
+        row.columns("name", "Vlad", "age")
+
+        then:
+        def exception = thrown(IllegalArgumentException)
+        exception.message == "Odd number of key/value pairs"
+    }
+
+    def "#columns throws IllegalArgumentException when odd object not a string"() {
```

Comment:
A few interesting scenarios maybe added: an empty array passed to `columns` method and `null` column name